code/notificationbuilder/src/main/java/com/adobe/marketing/mobile/notificationbuilder/internal/builders/BasicNotificationBuilder.kt

```diff
@@ -110,9 +111,11 @@ internal object BasicNotificationBuilder {
         context: Context,
         trackerActivityClass: Class<out Activity>?,
         broadcastReceiverClass: Class<out BroadcastReceiver>?,
-        dataMap: MutableMap<String, String>
+        intent: Intent
     ): NotificationCompat.Builder {
-        val basicPushTemplate = BasicPushTemplate(MapData(dataMap))
+        val extras = intent.extras
+            ?: throw NotificationConstructionFailedException("Intent extras are null, cannot fallback to basic notification.")
+        val basicPushTemplate = BasicPushTemplate(IntentData(extras, intent.action))
```

Comment:
Creating it this way will make `isFromIntent` to be true, even though the original data was from a map. Subsequently it would end up being created in the silent channel and won't honor sound settings